scripts/release/bug_bash.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021, Facebook
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Query the Top-Ten Bug Bashers
+
+This script will query the top-ten Bug Bashers in a specified date window.
+
+Usage:
+    ./scripts/bug-bash.py -t ~/.ghtoken -b 2021-07-26 -e 2021-08-07
+    GITHUB_TOKEN="..." ./scripts/bug-bash.py -b 2021-07-26 -e 2021-08-07
+"""
+
+import argparse
+from datetime import datetime, timedelta
+import operator
+import os
+
+# Requires PyGithub
+from github import Github
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-a', '--all', dest='all',
+                        help='Show all bugs squashed', action='store_true', default=False)
```

Comment:
```suggestion
                        help='Show all bugs squashed', action='store_true')
```

Just FYI, this is redundant with store_true.